apps/web/src/pages/auth/components/QuestionnaireForm.tsx

```diff
@@ -46,14 +47,14 @@ export function QuestionnaireForm() {
   >((data: ICreateOrganizationDto) => api.post(`/v1/organizations`, data));
 
   useEffect(() => {
-    if (currentUser?.environmentId) {
+    if (environment) {
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** ‏Temporary since next is broken :(